docs/src/api/class-browsercontext.md

```diff
@@ -852,6 +852,7 @@ Returns all open pages in the context.
 
 Routing provides the capability to modify network requests that are made by any page in the browser context. Once route
 is enabled, every request matching the url pattern will stall unless it's continued, fulfilled or aborted.
+It will consider the base URL if you set it via [`option: baseURL`] when creating the context.
```

Comment:
I don't think this tells me much. Can we move this to the description of the first "url" parameter, and give an example of how baseURL affects it? Same for other places.

Reply:
good idea with moving it, but not sure if it's better how its now with the wording.